theme/template-parts/content/content-profile.php

```diff
@@ -139,81 +154,114 @@
-                <a href="<?php echo $final_profile_url; ?>" aria-current="page" class="inline-block p-4 text-xperto-neutral-mid-1 hover:bg-xperto-orange-light-90 hover:text-xperto-orange <?php echo $active; ?>">About Me</a>
+                <a href="<?php echo $final_profile_url; ?>" aria-current="page"
+                    class="inline-block p-4 text-xperto-neutral-mid-1 hover:bg-xperto-orange-light-90 hover:text-xperto-orange <?php echo $active; ?>">About
+                    Me</a>
             </li>
             <li class="mr-2">
                 <?php $active = ($tabIndex === 1) ? 'xperto-tab-active' : ''; ?>
-                <a href="<?php echo $final_certificates_url; ?>" class="inline-block p-4 text-xperto-neutral-mid-1 hover:bg-xperto-orange-light-90 hover:text-xperto-orange <?php echo $active; ?>">Certificates</a>
+                <a href="<?php echo $final_certificates_url; ?>"
+                    class="inline-block p-4 text-xperto-neutral-mid-1 hover:bg-xperto-orange-light-90 hover:text-xperto-orange <?php echo $active; ?>">Certificates</a>
             </li>
             <li class="mr-2">
                 <?php $active = ($tabIndex === 2) ? 'xperto-tab-active' : ''; ?>
-                <a href="<?php echo $final_credentials_url; ?>" class="inline-block p-4 text-xperto-neutral-mid-1 hover:bg-xperto-orange-light-90 hover:text-xperto-orange <?php echo $active; ?>">Credentials</a>
+                <a href="<?php echo $final_credentials_url; ?>"
+                    class="inline-block p-4 text-xperto-neutral-mid-1 hover:bg-xperto-orange-light-90 hover:text-xperto-orange <?php echo $active; ?>">Credentials</a>
             </li>
 
         </ul>
         <?php
         // About Me Tab
-        if ($tabIndex === 0) :
-        ?>
+        if($tabIndex === 0):
+            ?>
             <div class="py-6 space-y-4">
-                <?php if ($mepr_user->ID === get_current_user_id()) : ?>
+                <?php if($mepr_user->ID === get_current_user_id()): ?>
                     <div class="w-full flex justify-end">
-                        <a href="<?php echo home_url('/account'); ?>" class="text-xperto-orange rounded-lg font-bold hover:bg-xperto-orange-light-90 p-2">Edit</a>
+                        <a href="<?php echo home_url('/account'); ?>"
+                            class="text-xperto-orange rounded-lg font-bold hover:bg-xperto-orange-light-90 p-2">Edit</a>
                     </div>
-                <?php
+                    <?php
                 endif;
-
-                if (!empty($profile)) : ?>
-                    <div class="w-full">
-                        <?php if (array_key_exists('mepr_about', $profile)) { ?>
-                            <?php echo $profile['mepr_about']; ?>
-                        <?php } ?>
+                ?>
+                <div class="w-full flex flex-wrap px-9">
+                    <?php if(!empty($profile) && isset($profile['mepr_about'])): ?>
```

Comment:
Are you able to handle the users with only about details?
Like no phone number? etc.
Previous code has high priority if "about" is filled in. In this code, the "about" will never show at all.